src/oop/object_interactions/hard/q4/QuestionText.md

```diff
@@ -0,0 +1,44 @@
+# oop-object_interactions-hard-q4
+
+- Java question
+
+## Question Text
+
+Some data should be encrypted, however to do so you need a key to base the encryption off of. Design part of a `CryptBox`
+class using the provided class components, and create a `crossEncrypt` method that takes another `CryptBox` object and 
+uses that object's key to encrypt the other object's data.
+
+### CryptBox Class:
+
```

Comment:
This is a hard level assignment so I guess they can write the whole object. If you want them to write it partially their tests should be in setup(). But here its better to just ask them to write it all.

Reply:
due to meeting, switching this to parsons instead